docs/reference/run.md

```diff
@@ -94,6 +94,20 @@ You can reattach to a detached container with `docker`
 [*attach*](/reference/commandline/attach). If you choose to run a
 container in the detached mode, then you cannot use the `--rm` option.
 
+When starting a container in detached mode, one needs to understand that, by
```

Comment:
@dharmit ahhh, ok...so you need two sentences:

This succeeds in starting the `nginx` service inside the container. However, it fails the detached container paradigm in that, the root process (`service nginx start`) returns and the detached container stops as designed. As a result, the `nginx` service is started but could not be used.